docs/_assets/variables.css

```diff
@@ -15,6 +15,9 @@ html {
   --footer-background: rgba(0, 0, 0, 0.1);
 
   --text-color: black;
+
+  /* search colors*/
+  --rocket-search-highlight-color: #2c3e50;
 }
 
 html.dark {
```

Comment:
To my knowledge we don't support dark mode for lion atm. All these values look like original rocket values, please remove them all, instead of adding one for the search.